JMS/com/ibm/mq/samples/jms/JmsResponse.java

```diff
@@ -118,6 +118,7 @@ private static void replyToMessage(JMSContext context, Message receivedMessage,
                 // can get the message off the temp reply queue
                 producer.setDeliveryMode(DeliveryMode.NON_PERSISTENT);

```

Comment:
Should the delivery mode be set to persistent since the change is to make the message processing transactional?